cmd/exo/cmd/sos_headers.go

```diff
@@ -172,14 +172,15 @@ var sosRemoveHeadersCmd = &cobra.Command{
 		}
 
 		objInfo.Metadata["content-type"] = []string{objInfo.ContentType}
+		objInfo.Metadata["x-amz-metadata-directive"] = []string{"REPLACE"}
```

Comment:
I'd use the appropriate API of `net.http.Header`

https://golang.org/pkg/net/http/#Header